data/json/recipes/recipe_others.json

```diff
@@ -2087,19 +2087,6 @@
     "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "forge", 150 ], [ "oxy_torch", 30 ] ] ],
     "components": [ [ [ "steel_chunk", 160 ], [ "steel_lump", 40 ], [ "frame", 4 ], [ "hdframe", 2 ] ] ]
   },
-  {
-    "type": "recipe",
```

Comment:
Is there going to be a craftable crucible to replace the recipe that was deleted here, or is this just going to be something you can only scavenge for?

Reply:
No, crucible_clay is meant to be sole craftable crucible